platemate/util/views.py

```diff
@@ -14,3 +16,18 @@ def turk_balance(request):
             return HttpResponse("OK: %s" % balance, status=200)
     except:
         return HttpResponse("Balance check failed.", status=503)
+
+def ping(request):
+    template = get_template('util/ping.html')
+    platemate_processor_running = False
+    try:
+        platemate_processor_status_output = subprocess.check_output('systemctl status platemate-processor.service', shell=True)
+        platemate_processor_running = "Active: active (running)" in platemate_processor_status_output
+    except:
+        platemate_processor_running = False
+    everything_ok = platemate_processor_running #other things can be added later
+    context = {
+        'platemate_processor_running' : platemate_processor_running,
+        'everything_ok' : everything_ok
+    }
```

Comment:
You've crept back into some massive variable names here...

How about `processor_up` and `processor_status`? The `platemate` part is rather redundant I think.